monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -470,15 +473,16 @@ def plan_and_process(
         if not no_pp:
             self.preprocess(c, n_proc, overwrite_plans_name, verbose)
 
-    def train_single_model(self, config: Any, fold: int, gpu_id: int = 0, **kwargs: Any) -> None:
+    def train_single_model(self, config: Any, fold: int, gpu_id: int | str | tuple = 0, **kwargs: Any) -> None:
```

Comment:
How about list as input? Can we also support that and then use `ensure_tuple`?

Reply:
Please also change the doc-string in L103:

https://github.com/Project-MONAI/MONAI/blob/b5dccbe020fb37d3f3dd146767f0ddfc20f68061/monai/apps/nnunet/nnunetv2_runner.py#L103

Reply:
This particular aspect presents a challenge due to the way Google Fire parses input strings into various formats. For instance, "1,2" will yield a tuple, while "1,2,3" will produce a string. This explains the inclusion of different type options within the arguments.

Reply:
I follow the example ("Argument Parsing") in the [python fire guide](https://google.github.io/python-fire/guide/).
I did find the difference. 
- "1,2" and "1,2,3" both produce tuple.
- "'1,2,3'" and '"1,2,3"' both produce str.

Would it be okay if I can get a snippet from you to re-produce the difference?

Reply:
@dongyang0122

Reply:
Also doc-string L478 needs to be updated. 

https://github.com/Project-MONAI/MONAI/blob/b5dccbe020fb37d3f3dd146767f0ddfc20f68061/monai/apps/nnunet/nnunetv2_runner.py#L478

Reply:
Thanks @dongyang0122 , Please feel free to either add or not add `list` as supported type. But the other two comments about the docstring would be necessary.

Reply:
I have included `list` as a supported type.

Reply:
Hi @dongyang0122 , here is what I see in GitHub review page. Did I miss something?

![image](https://user-images.githubusercontent.com/18563433/236610723-a33b9966-ffab-49a8-b2cb-cac0db9b21c8.png)

Reply:
I have introduced support for tuple, list and integer types.

Reply:
Thanks!